provisioning_templates/PXELinux/kickstart_default_pxelinux.erb

```diff
@@ -12,6 +12,8 @@ oses:
 <%
   major = @host.operatingsystem.major.to_i
   mac = @host.provision_interface.mac
+  proxy_uri = host_param('http-proxy') ? "http://#{host_param('http-proxy')}:#{host_param('http-proxy-port')}" : nil
```

Comment:
What if `host_param('http-proxy-port')` is nil/empty?

Reply:
I see, what you mean...